src/RecoveryServices/RecoveryServices.Backup.Management.Sdk/Generated/Models/IaasVMRecoveryPoint.cs

```diff
@@ -188,6 +188,5 @@ public IaasVMRecoveryPoint(string recoveryPointType = default(string), System.Da
         /// </summary>
         [JsonProperty(PropertyName = "recoveryPointProperties")]
         public RecoveryPointProperties RecoveryPointProperties { get; set; }
-
```

Comment:
Please revert this change as generated sdk is not allowed to change manually.

Reply:
reverted